src/qint.c

```diff
@@ -197,6 +229,19 @@ QINT_API size_t qint_decode4(BufferReader *br, uint32_t *i, uint32_t *i2, uint32
   return total + 1;
 }
 
+QINT_API size_t qint_decode32_64pair(BufferReader *br, uint32_t *n32, uint64_t *n64) {
```

Comment:
I really don't like this hack. If you want you can make this a wapper to decode3 and merge two of the 32 bit ints to one 64 bit.